script.js

```diff
@@ -1,24 +1,33 @@
-const openNav = () => document.getElementById("sidebar").style.width = "250px";
-
-const closeNav = () => document.getElementById("sidebar").style.width = "0";
-
-const toastToggle = id => {
-  var toast = document.getElementById(id);
-  if (toast.style.display === "none") {
```

Comment:
assign toast. style in a variable and use that variable

Reply:
As it is used here only that way not  defining it Separately  to avoid unnecessary  Variable Declaration

Reply:
If one thing repeats more than once in your JS you should avoid it by declaring var we work on JS and React on DRY Principles i.e DRY (Don't Repeat Yourself).